plugins/imagebase/plugin.js

```diff
@@ -727,7 +727,7 @@
 		/**
 		 * @property {CKEDITOR.dom.element} wrapper An element created for wrapping the progress bar.
 		 */
-		this.wrapper = CKEDITOR.dom.element.createFromHtml( wrapperHtml || '<div class="cke_loader"></div>' );
+		this.wrapper = CKEDITOR.dom.element.createFromHtml( CKEDITOR.tools.htmlSafeByReview(wrapperHtml || '<div class="cke_loader"></div>', 'safe const') );
```

Comment:
Is `wrapperHtml` a constant?

Reply:
`wrapperHtml` is set by the developer. It is used as a way to allow custom loader htmls for image uploading.

Reply:
Yep updated justification